truss/remote/baseten/api.py

```diff
@@ -184,18 +183,19 @@ def get_model_by_id(self, model_id: str):
             model(id: "{model_id}") {{
                 name
                 id
-                primary_version{{
```

Comment:
`get_model_by_id` changes should also be safe, since this was the only callsite: https://github.com/basetenlabs/truss/blob/8901a0121dda36b822fbabe62f8a172ba72c6c6e/truss/remote/baseten/remote.py#L104-L106